manifests/mtls-cert.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: cert-manager.io/v1
```

Comment:
Do we need to adjust the expiration for this cert?

Reply:
Its an example cert used for integration testing, not included as part of the actual chart.

Reply:
Thanks